_pytest/python.py

```diff
@@ -43,6 +44,14 @@ def _format_args(func):
     def _format_args(func):
         return inspect.formatargspec(*inspect.getargspec(func))
 
+if  sys.version_info[:2] == (2, 6):
```

Comment:
@RonnyPfannschmidt @hpk42 how about creating a `_pytest._compat` module to host this kind of "compatibility workaround" into a single place? There are other cases through the code where we do stuff differently based on python version. This would make simpler to remove code once support for a python version is dropped.

(this is out of the scope of this PR, just thought I would take this opportunity to mention it)